test/Feature/HLSLLib/tanh.32.test

```diff
@@ -0,0 +1,65 @@
+#--- source.hlsl
+
+StructuredBuffer<float4> In : register(t0);
+
+RWStructuredBuffer<float4> Out : register(u1);
+
+[numthreads(1,1,1)]
+void main() {
+  Out[0] = tanh(In[0]);
+  float4 Tmp = {tanh(In[1].xyz), tanh(In[1].w)};
+  Out[1] = Tmp;
+  float4 Tmp2 = {tanh(In[2].xy), tanh(In[2].zw)};
+  Out[2] = Tmp2;
+}
+
+
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
+Buffers:
+  - Name: In
+    Format: Float32
+    Stride: 16
+    Data: [ -0, 0, 1, -1, -0, 0, 1, -1, -0, 0, 1, -1]
+  - Name: Out
+    Format: Float32
+    Stride: 16
+    ZeroInitSize: 48
+  - Name: ExpectedOut # The result we expect
+    Format: Float32
+    Stride: 16
+    Data: [ -0.0, 0.0, 0.761594, -0.761594, -0.0, 0.0, 0.761594, -0.761594, -0.0, 0.0, 0.761594, -0.761594]
+Results:
+  - Result: Test1
+    Rule: BufferFloatEpsilon
+    Epsilon: 0.0008
```

Comment:
Can we drop this epsilon?

Reply:
I don't know, why would we want to do that?  This is apparently the tolerance DXC finds acceptable.

Reply:
If the tolerance is a copy of DXC, then it's fine.
I just expected there to be a lower epsilon for 32 bit compared to 16 bit, but they're the same.

Reply:
Yeah DXC had the same tolerance for both half and float.